prime-router/src/main/kotlin/fhirengine/engine/FHIREngine.kt

```diff
@@ -129,4 +131,14 @@ abstract class FHIREngine(
         actionLogger: ActionLogger,
         actionHistory: ActionHistory
     )
+
+    /**
+     * This value is used to delay the next step in the pipeline from processing the event in order to make sure
+     * that the requisite DB transaction has been committed.  This is only configured for production so as not to
+     * arbitrarily slow down staging or local development
+     *
+     * It should be removed as part of implementing https://github.com/CDCgov/prime-reportstream/issues/10638
+     */
+    val queueVisibilityTimeout: Duration =
+        if (Environment.get() == Environment.PROD) Duration.ofMinutes(5) else Duration.ZERO
```

Comment:
Five is a conservative number based on the timings seen in the logs, but since this is a bandaid conservative seemed better